### PR TITLE
Added all function comments

### PR DESCRIPTION
**Pull request by peterBrxwn**

### CODE REVIEW

File: Countly.d.ts [hunk 1]
```diff
@@ -554,11 +776,62 @@ declare module "countly-sdk-react-native-bridge" {
...
+     * @param {function} callback function to be called after fetching values.
+     * @return {string | void} error message or void
+     */
     export function remoteConfigUpdate(callback: CountlyCallback): string | void;
+
+    /**
+     *
+     * Replace specific Remote Config key value pairs with new values from server.
+     *
+     * @param {[string]} keyNames array of keys to replace.
```
Comment (arifBurakDemiray) [inline Countly.d.ts:793]: array docs like that? is not it "string[]"

File: Countly.js [hunk 2]
```diff
@@ -1734,6 +2030,13 @@ Countly.clearAllTraces = function () {
...
 };
 
+/**
+ *
+ * End a custom trace.
+ *
+ * @param {string} traceKey name of trace
+ * @return {string | void} error message or void
+ */
 Countly.endTrace = function (traceKey, customMetric) {
```
Comment (arifBurakDemiray) [inline Countly.js:2040]: what about other param?

**Review by arifBurakDemiray — CHANGES_REQUESTED** [review]
I do not think return doc required for the only void ones

**Comment by peterBrxwn** [comment]
[[RN] Add missing "return value" comments and missing function comments in general](https://trello.com/c/1zFAw6bX/1367-rn-add-missing-return-value-comments-and-missing-function-comments-in-general)

**Review by arifBurakDemiray — APPROVED** [review]

**Review by turtledreams — APPROVED** [review]

**Pull request merged**